src/v2/Components/NavBar/Menus/MobileNavMenu/InboxNotificationCount.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react"
+import { graphql } from "relay-runtime"
+import cookie from "cookies-js"
+import { isServer } from "lib/environment"
+import styled from "styled-components"
+
+import {
+  InboxNotificationCountQuery,
+  InboxNotificationCountQueryResponse,
+} from "v2/__generated__/InboxNotificationCountQuery.graphql"
+import { SystemContext } from "v2/Artsy/SystemContext"
+import { SystemQueryRenderer as QueryRenderer } from "v2/Artsy/Relay/SystemQueryRenderer"
+
+import { Flex, Sans, color } from "@artsy/palette"
+
+const FloatingDot = styled(Flex)`
+  background-color: ${color("purple100")};
+  border-radius: 50%;
+  width: 22px;
+  height: 22px;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 16px;
+  left: 19px;
+`
+
+const getConversationCount = () =>
+  (!isServer && cookie.get("conversation-count")) || 0
+
+const updateConversationCache = (conversationCount?: number) => {
```

Comment:
Looks like there is opportunity to DRY in 3 places by moving `getConversationCount` and `updateConversationCache` to a helper.

Reply:
Good call!!